Makefile

```diff
@@ -1,6 +1,6 @@
 # Define here list of available make commands.
 .PHONY: default pull up stop down clean exec exec\:wodby exec\:root drush \
-prepare install \
+prepare install features\:owner \
```

Comment:
It is a good practice to list the commands here in the same order as they are defined later in this file; please move `features\:owner` to the end of this list.

Reply:
It is corrected.